.github/workflows/lint.yaml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches:
       - main
-    tags:
```

Comment:
Won't this mean a change in behavior?

Reply:
It doesn't seem to be used as we use release branches 🤷 
Restored it.